scaleway/resource_domain_record_test.go

```diff
@@ -308,39 +386,6 @@ func TestAccScalewayDomainRecord_HTTPService(t *testing.T) {
 		ProviderFactories: tt.ProviderFactories,
 		CheckDestroy:      testAccCheckScalewayDomainRecordDestroy(tt),
 		Steps: []resource.TestStep{
-			{
```

Comment:
we should keep this test for dynamic record: `http_service`

Reply:
Test just added